charts/argo-cd/Chart.yaml

```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: Support ability to set .Values.namespaceOverride
```

Comment:
Can you drop the existing two lines? We only need your change here

Reply:
ah, well spotted! :)

Reply:
done :)